src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.dao.FruitsDaoCsvImpl;
+import core.basesyntax.dao.ReportDao;
+import core.basesyntax.dao.ReportDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FruitsService;
+import core.basesyntax.service.FruitsServiceImpl;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.OperationStrategyImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.operation.BalanceOperationHandler;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.PurchaseOperationHandler;
+import core.basesyntax.service.operation.ReturnOperationHandler;
+import core.basesyntax.service.operation.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String FROM_FILE_NAME = "src/main/resources/fromFile.csv";
+    private static final FruitsDao FRUITS_DAO = new FruitsDaoCsvImpl();
+    private static final ShopService shopService = new ShopServiceImpl();
+    private static final ReportDao reportDao = new ReportDaoImpl();
+
+    public static void main(String[] args) {
+        List<FruitTransaction> fruitTransactions = FRUITS_DAO.readDataFromFruitsCsv(FROM_FILE_NAME);
+
+        Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(Operation.BALANCE, new BalanceOperationHandler());
+        operationHandlerMap.put(Operation.SUPPLY, new SupplyOperationHandler());
+        operationHandlerMap.put(Operation.PURCHASE, new PurchaseOperationHandler());
+        operationHandlerMap.put(Operation.RETURN, new ReturnOperationHandler());
+
+        OperationStrategy operationStrategy = new OperationStrategyImpl(operationHandlerMap);
+
+        FruitsService fruitsService = new FruitsServiceImpl(operationStrategy);
+
+        Map<String, Integer> processTransactions =
+                fruitsService.processTransactions(fruitTransactions);
```

Comment:
```suggestion
        fruitsService.processTransactions(fruitTransactions);
```
you should have storage class and make changes in that storage